gm4_metallurgy/data/gm4_metallurgy/functions/casting/create_mould.mcfunction

```diff
@@ -2,6 +2,8 @@
 # run from check_mould_creation
 
 summon vex ~.45 ~-0.845 ~.65 {CustomName:"\"gm4_sand_ring\"",Tags:["gm4_sand_ring","gm4_new_sand_ring"],Attributes:[{Name:"generic.max_health",Base:0.25}],ArmorItems:[{},{},{},{id:"minecraft:player_head",Count:1,Damage:3,tag:{CustomModelData:23,SkullOwner:{Id:[I;140191582,-1151712444,-1271441790,1896239124],Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE0Nzk4NTI1MDM5MzksInByb2ZpbGVJZCI6Ijk4NWIyNzVlYmI1YTQzNDRiNDM3Njg5NTI4NjNhNjNmIiwicHJvZmlsZU5hbWUiOiJTcGFya3MiLCJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvMWZhYmExYzg3NTFjYWIzZjEyZjJlODgwMTRjZjRlYmY2N2ZjNWQ3NGNmY2U5YmUyNTI1YWQ0OWE5MjFkY2YyIn0sIkNBUEUiOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS81YzNjYTdlZTJhNDk4ZjFiNWQyNThkNWZhOTI3ZTYzZTQzMzE0M2FkZDU1MzhjZjYzYjZhOWI3OGFlNzM1In19fQ=="}]}}}}],NoAI:1,Silent:1,ActiveEffects:[{Id:14b,Amplifier:1b,Duration:2147483647,ShowParticles:0b}],Invunerable:1,PersistenceRequired:1}
+summon armor_stand ~.65 ~-0.545 ~.5 {Small:1,NoGravity:1,Marker:1,Invisible:1,Silent:1,DisabledSlots:2039552,Tags:["gm4_no_edit","gm4_sand_ring_display"],Fire:200000,CustomName:'"gm4_sand_ring_display"',HandItems:[{id:"minecraft:stone_button",Count:1b},{}],Pose:{RightArm:[0f, 0f, 0f]}}
+execute as @e[type=armor_stand,tag=gm4_sand_ring_display,limit=1,sort=nearest,distance=..1.2] run data modify entity @s HandItems[0].tag.CustomModelData set from entity @e[type=vex,tag=gm4_sand_ring,limit=1,distance=..1.2] ArmorItems[3].tag.CustomModelData
```

Comment:
I don't know if this would make a huge difference but as moulds are snapped to the block grid using
`align xyz <...> @e[dx=0<...>` might be safer than `distance=..1.2` and would allow getting rid of the sorting.
However, having to align the commands might be more of a hassle than it is worth and/or might add extra lag when a new execute command needs to be added.